geomesa-blobstore/geomesa-blobstore-core/src/main/scala/org/locationtech/geomesa/blob/core/handlers/WKTFileHandler.scala

```diff
@@ -0,0 +1,18 @@
+package org.locationtech.geomesa.blob.core.handlers
+
+import java.io.File
+import java.util
+
+import org.locationtech.geomesa.utils.text.WKTUtils
+
+import scala.collection.JavaConversions._
+
+class WKTFileHandler extends AbstractFileHandler {
+  override def canProcess(file: File, params: util.Map[String, String]) = {
+    params.contains("wkt")
+  }
+
+  override def getGeometryFromParams(params: util.Map[String, String]) = {
+    Option(WKTUtils.read(params("wkt")))
```

Comment:
Sure.  My main goal was to bang out the AbstractFileHandler and two quick examples.  

As a note, I'm leaving the WKTFileHandler in the core project since it doesn't require any dependencies and can serve a quick example class.